### PR TITLE
reformatting ipynb into .py scripts - Experiment B

### PR DESCRIPTION
**Pull request by mtchibozo**

### CODE REVIEW

**Comment by mtchibozo** [comment]
Great work Zijing, thank you!

- Could you please add a benchmark.py file to apply the same code from train.py to the different models we compared the ResNets to (XGBoost, SVM, LDA, Random Forests and MLP)? You can reuse parts of the experiments_resnet_model notebook which called each of these models.

- If you are able to load the history dataframe of one of our models from the drive (regularization = 4.5\ \times\ {10}^{-3}), could you please run the plotting function on them to generate the different accuracy & loss plots? This will allow researchers to view the same figures we show in the paper.

**Comment by mtchibozo** [comment]
Changes transferred to feature/packageSetup.
